ReactWindows/ReactNative.Shared/UIManager/Events/TouchEventType.cs

```diff
@@ -40,5 +40,10 @@ public enum TouchEventType
         /// Pointer exited event type.
         /// </summary>
         Exited,
+
+        /// <summary>
+        /// Pointer move event type.
+        /// </summary>
+        PointerMove
```

Comment:
`PointerMove` doesn't necessarily differentiate from `Move`, but `MoveWithoutStart` seems too verbose. `PointerMove` is probably fine (I can't think of anything better.